feature-libs/organization/administration/styles/_list.scss

```
@@ -261,6 +261,10 @@
         @include visible-focus();
         outline-offset: -2px;
       }
+
+      & td cx-toggle-link-cell a.invisible-focus-border {
```

definitely this long selectors needs to go. use the classname only
```suggestion
     .invisible-focus-border {
```

---

thanks for suggestion. Correction is done.